src/ios/GeolocationPlugin.m

```diff
@@ -0,0 +1,314 @@
+            const char *sql_table = "CREATE TABLE GPSTable(_id INTEGER PRIMARY KEY NOT NULL, latitude TEXT, longitude TEXT, altitude TEXT, timerInterval TEXT, region TEXT);";
+            
+            if (sqlite3_exec(contactDB, sql_table, NULL, NULL, &errMsg) != SQLITE_OK) {
+                NSLog(@"Failed to create table");
+            }
+            sqlite3_close(contactDB);
+        }
+        else{
+            NSLog(@"Failed to open/create databese");
+        }
+    }
+}
+
+-(void)InsertData{
+    sqlite3_stmt    *statement;
+    
+    const char *dbpath = [dataBasePath UTF8String];
+    
+    if (sqlite3_open(dbpath, &contactDB) == SQLITE_OK)
+    {
+        NSString *insertSQL = [NSString stringWithFormat:@"INSERT INTO GPSTable values(null, '%+.6f', '%.6f', '%.6f', '%@', '%@')", location.coordinate.latitude, location.coordinate.longitude, location.altitude, DateFormatter, EnterRegion];
+        
+        //NSString *insertSQL = [NSString stringWithFormat:@"INSERT INTO GPSTable values(null, '%.6f', '%.6f', '%.6f', '%@', '%@')", 15.12598, 524.12368, 15.6665, @"20160208T26:15:15", @"YES"];
+        
+        const char *insert_stmt = [insertSQL UTF8String];
+        
+        sqlite3_prepare_v2(contactDB, insert_stmt, -1, &statement, NULL);
+        if (sqlite3_step(statement) == SQLITE_DONE)
+        {
+            NSLog(@"Success to add contact");
+        } else {
+            NSLog(@"Failed to add contact");
+        }
+        sqlite3_finalize(statement);
+        sqlite3_close(contactDB);
+    }
+}
+-(void)MakeJSON_Make{
+    JSONArray = [[NSMutableArray alloc] init];
+    if (sqlite3_open([dataBasePath UTF8String], &contactDB) == SQLITE_OK) {
+        NSString *sql_makeJOSN = [NSString stringWithFormat:@"SELECT * FROM GPSTable"];
+        sqlite3_stmt *statement;
+        
+        if (sqlite3_prepare_v2(contactDB, [sql_makeJOSN UTF8String], -1, &statement, NULL)==SQLITE_OK) {
+            if (sqlite3_step(statement) == SQLITE_ROW) {
+                
+                while (sqlite3_step(statement) == SQLITE_ROW) {
+                    
+                    //NSLog(@"{timestamp: '%s', source: \"iOS\", gps: {latitude: %s, longitude: %s, altitude: %s, timerInterval %s}, region: {\"identifier\": \"%s\"}}", sqlite3_column_text(statement, 4),  sqlite3_column_text(statement, 1), sqlite3_column_text(statement, 2), sqlite3_column_text(statement, 3), sqlite3_column_text(statement, 4), sqlite3_column_text(statement, 5), sqlite3_column_text(statement, 6));
+                    
+                    
+                    JSONString = [NSString stringWithFormat:@"{timestamp: '%s', source: \"iOS\", gps: {latitude: %s, longitude: %s, altitude: %s, timerInterval %s}, region: {\"identifier\": \"%s\"}},", sqlite3_column_text(statement, 4),  sqlite3_column_text(statement, 1), sqlite3_column_text(statement, 2), sqlite3_column_text(statement, 3), sqlite3_column_text(statement, 4), sqlite3_column_text(statement, 5), sqlite3_column_text(statement, 6)];
```

Comment:
the JSON only contains what is supposed to be in the `data` attribute. please compare to android

```
{ 
"choice" : ["scan"],
  "data" : { /* this is as per your code */},
"comment" : "test",
"poll" : "/api/v1/polls/poll/rfscan/"
}
```